frontend/pages/blockly.html

```diff
@@ -340,11 +341,19 @@
     <category id="catFunctions" custom="PROCEDURE" name="Functions"></category>
     <sep></sep>
     <category id="shadow" name="Shadow Robot" colour="306">
-        <category name="Grasping">
+        <category name="Initialise">
+          <block type="initialise"></block>
+        </category>
+        <category name="Hand">
           <block type="grasp"></block>
         </category>
         <category name="Arm">
           <block type="arm"></block>
+          <block type="pose"></block>
```

Comment:
I think you can put more than one block per file (so that you don't have to edit this html each time you add a new function).